inscriptions/ISic000333.xml

```diff
@@ -185,24 +190,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141163" xml:lang="la">
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#MH">
                 <ab>
-                    <lb n="1"/>
-                    Arruntia Laudic
-                    <supplied reason="lost">e</supplied>
-                    <lb n="2"/>sibi et Aristoni<lb n="3"/>I
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <supplied reason="lost">s</supplied>uo
-                    <lb n="0"/>
-                    <gap reason="lost" quantity="1" unit="line"/>
+                    <lb n="1"/> Arruntia <g ref="#interpunct">·</g> Laudic<supplied reason="lost">e</supplied>
+                    <lb n="2"/>sibi <g ref="#interpunct">·</g> et <g ref="#interpunct">·</g> Aristoni
+                    <lb n="3"/>I <gap reason="lost" quantity="9" unit="character" precision="low"/>vo          
```

Comment:
the 'I' should probably go in `<orig>` to mark letters clear but of which no sense can be made.

also, ideally lose the initial space before Arruntia in line 1.